src/i_sound.c

```diff
@@ -549,13 +549,21 @@ void I_InitSound(void)
     ALCdevice *device;
     ALCcontext *context;
 
-    if (nosfxparm)
+    if (nosfxparm && nomusicparm)
     {
         return;
     }
 
     printf("I_InitSound: ");
 
+#ifdef WOOFDATADIR
+    {
+        const char conf_env[] = "share/"PROJECT_SHORTNAME"/alsoft.conf";
```

Comment:
Please replace `share/PROJECT_SHORTNAME` with `WOOFDATADIR` here. Should we check for the file's existence before?

Reply:
> Should we check for the file's existence before?

No, not needed on Windows, it just fallback to default ("cubic").

Reply:
That's cool. I just thought, because we now report to use this file for config, but don't even check if it's available.

Reply:
Well, `openal-soft` doesn't show that it even parses the config or supports all platform-specific options. OpenAL generally assumes that everything is configured automatically. I suggest not to print anything 😄

Reply:
Alright, fine with me.